hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleContextImpl.java

```diff
@@ -611,6 +624,66 @@ private void dispatchSyntheticTxn(
         }
     }
 
+    private record ValidationResult(
+            @NonNull PreHandleResult.Status status, @NonNull ResponseCodeEnum responseCodeEnum) {}
+
+    private ValidationResult validate(
+            @NonNull final HandleContextVerifier verifier,
+            HederaFunctionality function,
+            TransactionBody txBody,
+            AccountID payer,
+            Key payerKey) {
+
+        PreHandleContextImpl preHandleContext;
```

Comment:
`HandleWorkflow.validate()` contains a few additional checks that need to be included here. Do you want to add them in a separate PR?

Reply:
Which checks exactly? I'm guessing the payer status check?

Reply:
Right. If a payer is defined, we must check his status and solvency.

In addition, we need to check if the child transaction is a duplicate. I guess we also need to check the time box.